### PR TITLE
docs(resources): add iJS London talk and slide link

### PR DESCRIPTION
**Pull request by FabianGosebrink**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our guidelines: https://github.com/ngrx/platform/blob/master/CONTRIBUTING.md#commit
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Documentation has been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Resources page does not show the talk I added with this PR.

## What is the new behavior?

Resources page does show the talk I gave at iJS London.

## Does this PR introduce a breaking change?

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: projects/ngrx.io/content/marketing/resources.json [hunk 1]
```diff
@@ -29,6 +29,12 @@
             "url":
               "https://www.youtube.com/playlist?list=PL8OUS498tQP3FFsZzULTGnbyIcILFjHd3",
             "rev": true
+          },
+          "International-Javascript-Conference-London-2019": {
```
Comment (wesleygrimes) [inline projects/ngrx.io/content/marketing/resources.json]: Just a small tweak here, need these to be lowercase.

```suggestion
          "international-javascript-conference-london-2019": {
```

File: projects/ngrx.io/content/marketing/resources.json [hunk 2]
```diff
@@ -29,6 +29,12 @@
...
+            "title": "State Management in Angular with NgRx Stores and Effects",
```
Comment (wesleygrimes) [inline projects/ngrx.io/content/marketing/resources.json]: Just want to denote where the talk took place.

```suggestion
            "title": "State Management in Angular with NgRx Stores and Effects at iJS London 2019",
```

Comment (FabianGosebrink) [reply projects/ngrx.io/content/marketing/resources.json]: Just added the proposed changes. Thank you, nice catch.

**Comment by wesleygrimes** [comment]
@FabianGosebrink Looks like the talk title is still missing location and date. Can you push that up? Thanks!

**Comment by timdeschryver** [comment]
@wesleygrimes I believe date and location isn't needed here. This is a change to our resources page and not our events page. (if this is what you mean by it)

**Comment by wesleygrimes** [comment]
> @wesleygrimes I believe date and location isn't needed here. This is a change to our resources page and not our events page. (if this is what you mean by it)

No, I just meant this:

```diff
- "title": "State Management in Angular with NgRx Stores and Effects",`
+ "title": "State Management in Angular with NgRx Stores and Effects at iJS London 2019",`
```

**Comment by FabianGosebrink** [comment]
Pushed :)

**Comment by brandonroberts** [comment]
Thanks @FabianGosebrink!

**Comment by FabianGosebrink** [comment]
Thank you for building such amazing stuff. Really appreciate it.

**Pull request merged**